src/lib/utils/language.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * @fileoverview Tools for language selection and validation.
+ * TODO(devnook): Extract isSupportedLocale to a common format.
```

Comment:
Would you mind creating a bug to track this and linking the bug in this comment?